### DISCUSSION TITLE
How to access the layout

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [qtile/qtile#3603] [Q&A] [answered]
Hello, 
I wanted to make a function, that would change the current layout settings whenever the specified app is launched. In order to do that I used `client_new` hook:

```@hook.subscribe.client_new
def func(new_window):
    if new_window.name=='Win_Name':
        #alter the layout here
```

My problem is that I have no idea how I could access the layout in order to use functions like `cmd_grow()` or `cmd_shrink()`
Thanks in advance

**Comment by elParaguayo** [comment, marked as answer]
Try this:
``` python
from libqtile import qtile

def func(new_window):
    if new_window.name=='Win_Name':
        layout = qtile.current_layout
        layout.cmd_grow()  # etc
```

**Reply by elParaguayo** [reply]
Or, if you used the `client_managed` hook:
``` python
def func(window):
    layout = window.group.layout
    layout.cmd_grow()  # etc
```

**Comment by mcnuggetsx20** [comment]
@elParaguayo Thanks man, works perfectly
